backends/xboxone/tunneling.md

```diff
@@ -0,0 +1,70 @@
+# Tunneling to a Xbox One Device on Another Network
```

Comment:
We could maybe consider merging them, but there are a few subtle differences beyond port numbers.  For example, for Tizen, you have to set up your port forwards differently to accommodate the Tizen Studio tools running in Docker.

I also have found in the past that many people do not thoroughly understand SSH tunnels, so I felt more comfortable about having very explicit documentation that didn't have any "fill in your port number for service X here" that would require people to cobble together info from multiple documents.  I did it this way in the hope that it would be easier to follow the instructions for most people.

It's also pretty low-entropy documentation.  SSH tunneling isn't likely to change much in structure or commands necessary.  So having most of the content duplicated between docs is much less likely to cause issues than in some other contexts.

All that said, I'm still open to the idea of merging them.  How about I file an issue to follow up on that concept at a later time?

Reply:
https://github.com/google/generic-webdriver-server/issues/13